src/material/core/tokens/_token-utils.scss

```diff
@@ -71,12 +71,21 @@ $_component-prefix: null;
 
 // Emits a slot for the given token, provided that it has a non-null value in the token map passed
 // to `use-tokens`.
-@mixin create-token-slot($property, $token) {
+@mixin create-token-slot($property, $token, $emit-fallback: false) {
```

Comment:
Should we comment that having `emit-fallback = true` would be for temporary workarounds?